glean-core/rlb/src/private/ping.rs

```diff
@@ -94,6 +94,63 @@ impl PingType {
         self.inner.submit(reason.map(|s| s.to_string()))
     }
 
+    /// Get the name of this Ping
+    /// Wrapper around glean::PingType::name
```

Comment:
Since these would [end up in rustdocs](https://docs.rs/glean/63.0.0/glean/private/struct.PingType.html), I'd prefer that these function comments stand alone and not mention struct members from a separate crate.